priv/graphql/schemas/admin/http_upload.gql

```diff
@@ -0,0 +1,7 @@
+"""
+Allow admin to generate upload/download URL for the file on user's behalf".
+"""
+type HttpUploadAdminMutation @protected{
+    "Allow admin to generate upload/download URLs for the file on user's behalf"
+    getUrl(domain: String!, filename: String!, size: Int!, contentType: String!, timeout: Int!): String
```

Comment:
I think this should return a more structured output, where both URL's would be separate in GraphQL.